airbyte-integrations/connectors/source-jira/source_jira/source.py

```diff
@@ -97,6 +97,10 @@ def check_connection(self, logger: AirbyteLogger, config: Mapping[str, Any]) ->
                 return False, "unknown project(s): " + ", ".join(unknown_projects)
             return True, None
         except (requests.exceptions.RequestException, ValidationError) as e:
```

Comment:
This should be divided properly to avoid unnecessary `if`, like this:

```python
except requests.exceptions.RequestException as request_error:
    message = " ".join(map(str, request_error.response.json().get("errorMessages", "")))
    return False, f"{message} {request_error}"
except ValidationError as validation_error:
    return False, validation_error
```

make sure the `ValidationError` doesn't go first in the execution order, if so - please change the order of operations.